app/views/nomination_mailer/nominations_reminder_3_days_left.html.erb

```diff
@@ -50,38 +52,34 @@
 </p>
 
 <p>
-  We expect to announce the final ballot in early <%= @hugo_ballot_pub_month %>, and the awards will be
+  We expect to announce the final ballot in <%= @hugo_ballot_pub_month %>, and the awards will be
   presented at <%= @worldcon_public_name %>, which will run from <%= @start_day_informal %> to <%= @end_day_informal %>, <%= @worldcon_year %>.
 </p>
 
 <p>
   The Hugo Awards are fan-run, fan-given, and fan-supported. We recommend that
   you nominate whatever works and creators you have personally read or seen that
-  were your favorites from <%= @worldcon_year_before %> and <%= (@retro_hugo_75_ago.to_i - 1).to_s %>
+  were your favorites from <%= @worldcon_year_before %>.
 </p>
 
 <p>
   Please contact <%= @email_hugo_help %> if you have difficulties accessing the
-  online ballot(s), or you have any other questions on the Hugo process.
+  online ballot(s), or you have any other questions on the Hugo Award process.
 </p>
 
 <p>
-  A printable version of the WSFS Constitution and Hugo Nominating ballot has
-  been included in the mailed-out version of Progress Report 2. You may also
-  download a printable ballot in
-  <a href=<%= @hugo_ballot_download_a4 %>>A4</a>,
-  or
-  <a href=<%= @hugo_ballot_download_letter %>>US letter</a>
-  sizes. The WSFS Constitution is also available online
-  <a href=<%= @wsfs_constitution_link %>>here</a>.
+  A printable version of the WSFS Constitution and Hugo Award Nominating ballot 
+  are available online 
+  <a href=<https://chicon.org/home/whats-happening/hugo-awards/>>here</a>, but
```

Comment:
That's fine as long as @wsfs_constitution_link can be set to https://chicon.org/home/whats-happening/hugo-awards/ since that's where both the printable ballot and WSFS constitution links live.

Reply:
Easy.